packages/pg/lib/connection.js

```diff
@@ -87,9 +106,9 @@ class Connection extends EventEmitter {
         }
       }
 
-      if (net.isIP(host) === 0) {
-        options.servername = host
-      }
+      // if (net.isIP(host) === 0) {
+      //   options.servername = host
+      // }
```

Comment:
Reminders me, does this PR work with IPs and hostnames?

Reply:
It did not, but fixed now, thanks for the catch